vital/types/protobuf/activity.proto

```diff
@@ -32,14 +32,14 @@ syntax = "proto2";
 
 package kwiver.protobuf;
 
-import "class_map.proto";
+import "activity_type.proto";
 import "object_track_set.proto";
 import "timestamp.proto";
 
 message activity {
   required int64 id = 1;
   required string label = 2;
-  optional class_map activity_type = 3;
+  optional activity_type activity_type_ = 3;
```

Comment:
Fair question? For `detected_object`, the corresponding field is `classifications`; I wonder if `activity` should do likewise? (Note: I am not responsible for that code :wink:; either the protobuf, or as it is in `activity` itself.)

Reply:
I think the extra underscore was to avoid confusion between the type "activity_type" and the variable that we're setting =3. I think renaming it to classifications is a good idea.

Reply:
I put the extra underscore there because the build was resolving the actual activity_type() method in activity to that variable in the .proto file. Renaming to classifications would absolutely solve this problem.

Reply:
Done (now "classifications").